### PR TITLE
feat:add a token and password encryption

### PR DESCRIPTION
**Pull request by luhaoling**

### CODE REVIEW

**Comment by luhaoling** [comment]
To pasword encryption，I add a new column to database.users.
Here is the sql:
`salt` longtext CHARACTER SET utf8mb4 COLLATE utf8mb4_0900_ai_ci NULL,

File: app/user/service/internal/biz/user.go [hunk 1]
```diff
@@ -1,28 +1,35 @@
...
+	FollowerCount   uint32    `gorm:"column:follower_count;not null;default:0"`
+	Avatar          string    `gorm:"column:avatar_url;not null;default:''"`
+	BackgroundImage string    `gorm:"column:background_image_url;not null;default:''"`
+	Signature       string    `gorm:"column:signature;not null;default:''"`
+	TotalFavorited  uint32    `gorm:"column:total_favorited;not null;default:0"`
+	WorkCount       uint32    `gorm:"column:work_count;not null;default:0"`
+	FavoriteCount   uint32    `grom:"column:favorite_count;not null;default:0"`
+	IsFollow        bool      `gorm:"-"`
+	Salt            string    `gorm:"column:salt"`
+	Created_at      time.Time `gorm:"column:created_at"`
```
Comment (intyouss) [inline app/user/service/internal/biz/user.go]: **Warning**
You should add them in data/user.go

**Explanation**
The biz layer serves the business, and the time field you defined, as well as the SALT, etc., are fields in the DATA layer that need to be defined in the User model, not in the response model.And I think updating and creating fields is unnecessary as there is no place to use it, don't go adding too much redundancy

**Comment by luhaoling** [comment]
Ok,I understand.I will alter the relevant code.

**Comment by luhaoling** [comment]
Ok,I understand.I will alter the relevant code.

**Pull request closed without merging**